Resources/Prototypes/Entities/Clothing/OuterClothing/misc.yml

```diff
@@ -95,7 +95,7 @@
     - WhitelistChameleon
 
 - type: entity
-  parent: ClothingOuterBaseToggleable
+  parent: [ ClothingOuterBaseToggleable,  ClothingOuterStorageBase ] # DeltaV - "Hoodie" Pockets
```

Comment:
inherit base winter coat so the temperature protection doesnt need to be copy pasted